use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -0,0 +1,491 @@
+onto the host as well as preparing a specially built Kata Containers® kernel 
+and custom Kata Containers rootfs.  
+
+## Helpful Links before starting
+
+[Intel QAT Engine](https://github.com/intel/QAT_Engine)
+
+[Intel QuickAssist Technology at 01.org](https://01.org/
+intel-quickassist-technology)
+
+[hIntel Device Plugin for Kubernetes](https://github
+.com/intel/intel-device-plugins-for-kubernetes/)
+
+[Intel QuickAssist Crypto Poll Mode Driver](https://dpdk-docs.readthedocs.io/
+en/latest/cryptodevs/qat.html)
+
+## Steps to enable Intel QAT in Kata Containers
+
+Some of the steps only have to be done once. Some have to be done every 
+reboot. Other steps are required only when the host kernel changes.
+
+## Script variables 
+
+The variables listed below must be set before running through the scripts. 
+These variables refer to locations to store modules and configuration files on
+ the host and as well as links to the drivers to use. Modify these as needed 
+ to point to updated drivers or different install locations.
+
+### Set environment variables (Every Reboot)
+
+```sh
+$ export QAT_CONF_LOCATION=~/QAT_conf
+$ export QAT_DRIVER_URL=https://01.org/sites/default/files/downloads/qat1.7.l.4.5.0-00034.tar.gz
+$ export QAT_DOCKERFILE=https://raw.githubusercontent.com/intel/intel-device-plugins-for-kubernetes/master/demo/openssl-qat-engine/Dockerfile
+$ export QAT_SRC=~/src/QAT
+$ export GOPATH=~/src/go
+$ export OSBUILDER=~/src/osbuilder
+$ export KATA_KERNEL_LOCATION=~/kata
+$ export KATA_ROOTFS_LOCATION=~/kata
+```
+
+## Prepare the Host (Clear Linux)
+
+The host could be a bare metal instance or a virtual machine. If using a 
+virtual machine, make sure that KVM nesting is enabled.  All of the 
+instructions below reference an Intel QAT.  Some of the instructions must be 
+modified if using a different Intel QAT device. You can identify the Intel QAT
+chipset by executing the following.
+
+### Identify which PCI Bus the Intel QAT card is on
+
+```
+$ for i in 0434 0435 37c8 1f18 1f19; do lspci -d 8086:$i; done
+b1:00.0 Co-processor: Intel Corporation C62x Chipset QuickAssist Technology (rev 04)
+	b3:00.0 Co-processor: Intel Corporation C62x Chipset QuickAssist Technology (rev 04)
+	b5:00.0 Co-processor: Intel Corporation C62x Chipset QuickAssist Technology (rev 04)
+```
+
+### Install necessary bundles for Clear Linux and disable kernel updates
+
+The Clear Linux installer generally installs the `os-core`, `os-core-update`, 
+and `openssh-server` bundle. Add the following minimum bundles including the 
+Linux kernel headers to complete the rest of this guide. The Linux kernel 
+headers are needed to compile the out of tree kernel modules to support Intel 
+QAT on the host. The `os-clr-on-clr` is a large development bundle that can be 
+installed instead to include everything below with additional items. This 
+guide installs the `4.19` LTS kernel from 2018 because the crypto modules are 
```

Comment:
I'd refer to the following for complete clarity:

> Linux LTS kernel version `4.19`

Reply:
Updated.